internal/pkg/cli/app_delete.go

```diff
@@ -93,42 +93,42 @@ func newDeleteAppOpts(vars deleteAppVars) (*deleteAppOpts, error) {
 		},
 		pipelineLister: deploy.NewPipelineStore(rg.New(defaultSession)),
 		sel:            selector.NewAppEnvSelector(prompter, store),
-		svcDeleteExecutor: func(svcName string) (executor, error) {
+		svcDeleteExecutor: func(appName, svcName string) (executor, error) {
 			opts, err := newDeleteSvcOpts(deleteSvcVars{
 				skipConfirmation: true, // always skip sub-confirmations
 				name:             svcName,
-				appName:          vars.name,
```

Comment:
is `vars.name` always going to be equal to the appname we pass through here in cases other than what this PR fixes?

Reply:
yes 👍